mockwebserver/src/main/java/okhttp3/mockwebserver/MockResponse.java

```diff
@@ -43,6 +43,9 @@
   private long bodyDelayAmount = 0;
   private TimeUnit bodyDelayUnit = TimeUnit.MILLISECONDS;
 
+  private long shutdownDelay = 1;
```

Comment:
Typically we just store milliseconds in the fields. The setters can take (duration, unit) and we can convert to milliseconds.